scripts/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Had the damnedest time getting `ts-node` to work right w/ these scripts. This special tsconfig along side `module-alias` and a special way of running the scripts (`ts-node --dir ./script <script>`) was the crazy specialness I had to bundle together. The `--dir` flag is important because that changes how `ts-node` looks for tsconfig files and without this tsconfig file it _definitely_ doesn't work, ha.

Reply:
Thoughts on just keeping as `.js` and using `// @ts-check`? Have found this pretty sufficient in various force code-paths that don't intersect main asset pipeline.